ui/tests/e2e/test_config.ts

```diff
@@ -0,0 +1,12 @@
+const URL_PREFIX = process.env.CI
```

Comment:
Can the environment variable directly specify the address?

Reply:
You mean pass the address direclty vis env? like this: `ADDRESS=xxx yarn test`?

Reply:
Yes. If I want to test the dashboard in PD, I need to pass the PD's address into it.

Reply:
Got it, ok, I will update it soon.

Reply:
done!